tools/myst/host/syscall.c

```diff
@@ -304,6 +305,9 @@ long myst_open_ocall(
     uid_t uid,
     gid_t gid)
 {
+    if (uid == UINT_MAX && gid == UINT_MAX)
```

Comment:
MUST be wrapped in #ifdef MYST_ENABLE_GCOV

Reply:
Added that, thank you for pointing this out